### PR TITLE
Tank adjustments

### PR DESCRIPTION
**Pull request by chizzy376**
## About The Pull Request
PR made by Bill
image
## Why It's Good For The Game
"Xenos won't die instantly" -Bill Torres
## Changelog
:cl: 
balance: changed tank's rocket from 100 sunder to 20
balance: changed tank's minigun ltaap penetration from 10 to 15
/:cl:

### CODE REVIEW

**Comment by Woker2004** [comment]
You do know tank is made out of paper right now? Anyways make the tank shell do at least 80 Sunder because you get limited amount and it is, I don't know, a fucking tank round, if your doing 20 sunder on hit just add a HE round that has bigger radius

**Comment by womk2004** [comment]
> ## Changelog
> 🆑 balance: changed tank's rocket from 100 sunder to 20
Also it is a Tank shell not a Tank Rocket

**Comment by SimplySmiley123** [comment]
> You do know tank is made out of paper right now? Anyways make the tank shell do at least 80 Sunder because you get limited amount and it is, I don't know, a fucking tank round, if your doing 20 sunder on hit just add a HE round that has bigger radius

Yeah, the tank, and by proxy, the APC seem to lately just fold the instant a xeno gets to touch 'em. Forcing a very passive playstyle with them if they want to survive for longer then 5 minutes after the LZ gates open.

And regarding tank shells I would love to see some variants to them, an anti-tank shell for ruining a target's armour and sundering them hard at cost of lack of AOE, and an HE round that deals less sunder but causes a larger explosion on impact.
Cuz frankly going from 100 to 20 on a _god damn tank shell_ is, kinda ridiculous in it's current state of being a glass cannon.

**Comment by Lumipharon** [comment]
>nerf pr buffs minigun

This sunder nerf seems quite extreme

**Comment by CVSthePharmacy** [comment]
hi chimmy, also no close this pr please, that is a massive nerf to an already nerfed tank, don't kick a dead horse man...

File: code/modules/projectiles/ammo_types/heavy_ammo.dm [hunk 1]
```diff
@@ -34,7 +34,6 @@
 /datum/ammo/bullet/minigun/ltaap
 	name = "chaingun bullet"
 	damage = 30
-	penetration = 10
```
Comment (TiviPlus) [inline code/modules/projectiles/ammo_types/heavy_ammo.dm:37]: set it explicitly instead of inheriting

**Review by TiviPlus — CHANGES_REQUESTED** [review]
otherwise ok

**Comment by TiviPlus** [comment]
> > nerf pr buffs minigun
> 
> This sunder nerf seems quite extreme

Pretty sure it was originally supposed to have 0 sunder

**Comment by TiviPlus** [comment]
Paper issues should be adressed seperately by actually standardizing the anti vehicle buffs instead of beign "lol lmao" kneejerk mech changes

File: code/modules/projectiles/ammo_types/heavy_ammo.dm (hunk 1 above)
Comment (chizzy376) [reply code/modules/projectiles/ammo_types/heavy_ammo.dm:37]: Done

**Comment by CVSthePharmacy** [comment]
don't merge these nerfs.... changing the title doesn't make it not a nerf.

**Pull request merged**